homeassistant/components/unifi/unifi_client.py

```diff
@@ -25,10 +25,12 @@
 CLIENT_BLOCKED = (WIRED_CLIENT_BLOCKED, WIRELESS_CLIENT_BLOCKED)
 CLIENT_UNBLOCKED = (WIRED_CLIENT_UNBLOCKED, WIRELESS_CLIENT_UNBLOCKED)
 WIRED_CLIENT = (WIRED_CLIENT_CONNECTED, WIRED_CLIENT_DISCONNECTED)
+WIRELESS_CLIENT_ROAMRADIO = "EVT_WU_RoamRadio"
```

Comment:
with the update to aiounifi, won't this be unnecessary or even conflict with aiounifi.events.WIRELESS_CLIENT_ROAMRADIO?

Reply:
Yes, but the bump in library won't happen for 0.109

Reply:
gotcha, makes sense!